mlflow/transformers/__init__.py

```diff
@@ -94,6 +94,8 @@
 _IMAGE_PROCESSOR_TYPE_KEY = "image_processor_type"
 _INFERENCE_CONFIG_BINARY_KEY = "inference_config.txt"
 _INSTANCE_TYPE_KEY = "instance_type"
+_INFERENCE_TASK_KEY = "inference_task"
+_METADATA_INFERENCE_TASK_KEY = "task"
```

Comment:
Shouldn't we use `inference_task` for consistency?

Reply:
We should, but Databricks Provisioned Throughput uses `metadata: { "task": ... }` for making optimizations, so IMO saving to the `task` field here would prevent the need from logging the model with both
```python
mlflow.transformers.save_model(
  ...,
  task="llm/v1/completions", # <- added in this PR
  metadata={"task": "llm/v1/completions"}, # <- currently required
)
```

Reply:
Makes sense, thanks! Could you add a comment describes that context for the future reference?